internal/logger/logger.go

```diff
@@ -98,12 +100,22 @@ func InitLogFile(logConfig config.LogConfig) error {
 func init() {
 	defaultLoggerFactory = &loggerFactory{
 		file:            nil,
+		format:          jsonFormat,
```

Comment:
Could we create a constant `DefaultLogFormat = jsonFormat` to be able to switch it in the future ?